DOCS-8828 enable inbound traffic prereq info (#285)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/install-prereqs.adoc

+
If your infrastructure does not meet the minimum hardware, operating system, and networking requirements, Runtime Fabric will not operate successfully.

> * You will need the assistance of your organization’s:

Reviewer: ```suggestion
* Work with the following teams in your organization:
```

> ** DevOps team to provision and manage the infrastructure

Reviewer: ```suggestion
** DevOps, to provision and manage the infrastructure
```

> ** Network team to whitelist ports and configure proxy settings

Reviewer: ```suggestion
** Network, to enable ports and configure proxy settings
```

> ** Security team to review for compliance and obtain security certificates

Reviewer: ```suggestion
** Security, to review for compliance and obtain security certificates
```

+
Working with these teams in your organization helps ensure a successful installation and avoids performance and density issues. 

> * Review the requirements in this topic and ensure that all prerequisites are met before installing Anypoint Runtime Fabric.

Reviewer: ```suggestion
* Review the provided requirements and ensure that all prerequisites are met before installing Anypoint Runtime Fabric.
```

+
The Runtime Fabric installation process fails if the minimum requirements are not met.

See xref:architecture.adoc[Runtime Fabric Architecture] for a description of development and production configurations.

If needed, contact your MuleSoft representative for assistance.

> == General Requirements

Reviewer: ```suggestion
== Runtime Fabric Installation Requirements
```

Knowledge in the following areas is required to perform a Runtime Fabric installation:

* Linux system commands
* Working in Linux with root access. Installation requires root access (permission to run the `sudo` command) on all servers in your environment
* If running on Amazon Web Services (AWS), working with Terraform
* Remote connection mechanisms, such as SSL and SSH

Ensure that you plan for:

* Creating separate Runtime Fabrics for production and non-production applications
* Creating separate Runtime Fabrics based upon environment network scope, such as DMZ versus private network 
* Ensuring that the Runtime Fabric default pod and services CIDR block do not interfere with the regions network

== Anypoint Platform Roles and Permissions
To succesfully use Anypoint Runtime Fabric, your Anypoint Platform account must have the following roles and permissions enabled:

> * Ability to use Anypoint Runtime Manager to deploy and manage applications
> * Ability to use Anypoint Access Management to manage permissions for Anypoint Platform users
> ** To use Anypoint Runtime Fabric, you must have the *Organization Administrators* role or the *Manage Runtime Fabrics* permission on the corresponding environments.
> ** Administrators need the *Manage Runtime Fabrics* permission at the organization level to be able to delete a Runtime Fabric.
> ** To deploy applications, you must have the *Exchange Contributors* role enabled for your Anypoint Platform account.

Reviewer: ```suggestion
* To use Anypoint Runtime Fabric, the *Organization Administrators* role or the *Manage Runtime Fabrics* permission on the corresponding environments.
* To delete Anypoint Runtime Fabric, the *Manage Runtime Fabrics* permission at the organization level
* To deploy applications, the *Exchange Contributors* role in your Anypoint Platform account
```

*** The subnet range used for Runtime Fabric in CIDR notation
*** If virtual machines are allowed, public IP addresses
*** How to gain shell access to each VM

== Inbound Traffic Enablement Requirements

Determine if inbound traffic should be enabled for Anypoint Runtime Fabric. This is based on the function of your organization's Mule applications. 

> *** If you plan to configure a TLS context using PEM or JKS certificates, you must have the necessary permissions for the Runtime Fabric environment you want to use for applications: Refer to xref:anypoint-security::configure-tls-context-rtf-lb.adoc[Configure a TLS Context for Anypoint Runtime Fabric] for details about the permissions needed.

Reviewer 2: With the Multiple Virtual Servers UI, no additional permissions are needed beyond Manage Runtime Fabrics (I think, I will check to make sure). Either way, that doc is specific to Secrets Manager which is no longer required for enabling inbound traffic.

Author: Fixed this.

*** The subnet range used for Runtime Fabric in CIDR notation
*** If virtual machines are allowed, public IP addresses
*** How to gain shell access to each VM

> * Determine if inbound traffic should be enabled for Anypoint Runtime Fabric. This is based on the function of your organization's Mule applications.

Reviewer: ```suggestion
* Based on the function of your organization's Mule applications, determine if inbound traffic should be enabled for Anypoint Runtime Fabric. 
```

+
If you plan to enable inbound traffic, evaluate your network's traffic to determine whether you need to configure dedicated nodes for the inbound load balancer. For additional information, refer to xref:architecture.adoc#network-architecture[Network Configuration] and xref:deploy-resource-allocation.adoc#internal-load-balancer[Internal Load Balancer].

> 

Reviewer: Is this level of detail nec. in this context?
